api/types/saml_idp_service_provider.go

```diff
@@ -21,15 +21,25 @@ import (
 	"fmt"
 
 	"github.com/gravitational/trace"
+	"golang.org/x/exp/slices"
 
 	"github.com/gravitational/teleport/api/utils"
 )
 
+const (
+	unspecifiedNameFormat = "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
```

Comment:
We have these constants already defined in https://github.com/gravitational/teleport.e/blob/master/lib/idp/saml/assertion.go#L22. I am working on followup PRs which will likely move most (if not all) of the constants defined in that file to this one. So for now, I decided to just redefine the needed constants here. Let me know if you think that is better done in this PR.

Reply:
As long as we don't end up in a permanent state where these constants are duplicated, I've no objections.

Reply:
Thanks. Noted and will make sure we don't end up that way!